packages/python/build.sh

```diff
@@ -48,13 +47,25 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" ac_cv_func_shm_open=yes"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" ac_cv_func_shm_unlink=yes"
 # Assume tzset() works
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" ac_cv_working_tzset=yes"
+# Disable tkinter module.
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" py_cv_module__tkinter=n/a"
```

Comment:
Don't we need that as a dependency for some other packages?

Reply:
https://github.com/termux/termux-packages/pull/24647#issuecomment-2859830798